### PR TITLE
Web/home/stories

### LINKED ISSUE #73: Implement homepage "Stories"

### PR DESCRIPTION
**Pull request by mingyokim**
:tickets: **Ticket(s)**: Closes #73

---

## :construction_worker: Changes

Implemented Hacker stories

## :thought_balloon: Notes

this was too much css for me rip

## :flashlight: Testing Instructions

make sure this is hella responsive!!!

### CODE REVIEW

File: web/components/home/Home.js [hunk 1]
```diff
@@ -133,8 +190,28 @@ class Home extends React.Component {
           </div>
         </div>
 
+        <div className={`${ROW_STYLE}`}>
+          <h2 className="fill-width margin-bottom-xxl">Hacker stories</h2>
+          <div className="flex jc-between row-stories scale-row-tablet">
+            {
+              ['spencer', 'olivia', 'avinash'].map(person => (
```
Comment (chamkank) [inline web/components/home/Home.js]: Love the use of map here, makes understanding the layout super easy 💯

File: web/components/home/Home.js [hunk 2]
```diff
@@ -133,8 +190,28 @@ class Home extends React.Component {
...
+                <div className="scale-width-tablet fill-height" key={`${person}`}>
+                  <div className="relative overflow-hidden">
+                    {getImage(`${person}-bg`, userToBg[person], 'fill-width pad-right-s')}
+                    {getImage(`${person}`, userToImg[person], 'fill-width overlay')}
+                    <div className="overlay scale-hide-full-screen scale-hover-disable-full-screen hover-show fill-width fill-height">
```
Comment (chamkank) [inline web/components/home/Home.js]: Should the hover effects be working right now?

Comment (mingyokim) [reply web/components/home/Home.js]: Yes, this PR is finalized.. is it not working for you? Also hover only works for 1900px or higher viewport width

Comment (chamkank) [reply web/components/home/Home.js]: Ah I see it now, looks good! Is there a reason why it's enabled only for >1900px viewport width?

Comment (mingyokim) [reply web/components/home/Home.js]: Cant fit the text inside the box

**Pull request merged**